server/@types/journeys/index.d.ts

```diff
@@ -15,8 +15,10 @@ declare namespace journeys {
   }
 
   export interface DateOfBirth {
-    isKnown: boolean
-    dateOfBirth?: Date
+    isKnown: 'Yes' | 'No'
```

Comment:
I wanted this to be an enum but for whatever reason when it compiles to javascript it can't reference it properly as `journeys.MyEnum`. To be investigated/fixed in another PR